perception/autoware_bevfusion/config/bevfusion_camera_lidar.param.yaml

```
@@ -11,6 +11,12 @@
     cloud_capacity: 2000000
     onnx_path: "$(var model_path)/bevfusion_camera_lidar.onnx"
     engine_path: "$(var model_path)/bevfusion_camera_lidar.engine"
+
+    # image backbone network (fusion model always uses separate image backbone)
+    image_backbone_onnx_path: "$(var model_path)/image_backbone.onnx"
+    image_backbone_engine_path: "$(var model_path)/image_backbone.engine"
+    image_backbone_trt_precision: fp16
+
```

I think we should directly point model's name in file path, therefore I suggest to add `bevfusion` prefix. CenterPoint does same way (suffix). Please populate this change in all the BEVFusion's code as well (schema, param name in cpp).
```suggestion
    # image backbone network (fusion model always uses separate image backbone)
    image_backbone_onnx_path: "$(var model_path)/bevfusion_image_backbone.onnx"
    image_backbone_engine_path: "$(var model_path)/bevfusion_image_backbone.engine"
    image_backbone_trt_precision: fp16
```